src/PluginForm/Onsite/PaymentMethodAddForm.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Drupal\commerce_heartland\PluginForm\Onsite;
+
+use Drupal\commerce_payment\PluginForm\PaymentMethodAddForm as BasePaymentMethodAddForm;
+use Drupal\Core\Form\FormStateInterface;
+
+class PaymentMethodAddForm extends BasePaymentMethodAddForm
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function buildCreditCardForm(array $element, FormStateInterface $form_state)
+    {
+        $plugin = $this->plugin;
+
+        $element['#attributes']['class'][] = 'heartland-form';
+
+        $element['#attached']['drupalSettings']['heartland'] = [
+            'publicKey' => $plugin->getPublicKey(),
+        ];
+        $element['#attached']['library'][] = 'commerce_heartland/secure-submit';
+        $element['#attached']['library'][] = 'commerce_heartland/form';
+
+        $element['number'] = [
+            '#type' => 'text',
+            '#required' => true,
+            '#validated' => true,
+            '#markup' => '<div class="form-group">
+                <label for="iframesCardNumber">Card Number:</label>
+                <div class="iframeholder" id="iframesCardNumber"></div>
+                </div>',
```

Comment:
Let's prefix all class names/IDs that are specific to our integration with `heartland-` (or something similar) to prevent any potential collisions with other integrations.